src/bin/busd.rs

```diff
@@ -44,7 +59,23 @@ async fn main() -> Result<()> {
 
     let args = Args::parse();
 
-    let mut bus = bus::Bus::for_address(args.address.as_deref()).await?;
+    let config_path = if args.system {
+        PathBuf::from("/usr/share/dbus-1/system.conf")
+    } else if let Some(config_path) = args.config {
+        config_path
+    } else {
+        PathBuf::from("/usr/share/dbus-1/session.conf")
+    };
+    eprintln!("reading configuration file {} ...", config_path.display());
```

Comment:
still using eprintln. Please use `tracing` methods to all logging.

Reply:
I'll switch these over, but I did keep these using `println!()` because it was consistent with the code that was already there

I assumed "bin" code that was outside of the "lib" part of the crate was intended to display output to users without all the spans and attributes that `tracing` provides :shrug:

Reply:
> I assumed "bin" code that was outside of the "lib" part of the crate was intended to display output to users without all the spans and attributes that `tracing` provides 🤷

Well sort of. It depends on what's being displayed. E.g the address should still be output directly to stdout when `--print-address` is passed. Later we want to also allow writing to an FD, like `dbus-daemon` does. Any logging messages, should go through `tracing`.